controller/work_item_type_group.go

```diff
@@ -74,10 +74,12 @@ func (c *WorkItemTypeGroupController) List(ctx *app.ListWorkItemTypeGroupContext
 // object for jsonapi.org specification
 func ConvertTypeGroup(request *http.Request, tg workitem.WorkItemTypeGroup) *app.WorkItemTypeGroupData {
 
+	spaceTemplateID := space.SystemSpace
```

Comment:
If I understand correctly this assignment of space to the space template is just for sake of getting space template language in?

Reply:
Yep